src/pages/Google.tsx

```diff
@@ -1,12 +1,7 @@
-import { Link } from 'react-router-dom';
-
 export default function Google() {
```

Comment:
@tangigouez We should think of a template page like "Avastar is currently developping this page focusing on the Facebook platform to help you ....

Reply:
Oui effectivement. Je vais réfléchir à ça 👍